src/TextArea/text-area-style.scss

```diff
@@ -1,4 +1,10 @@
-$red: #F03529;
+$red: #f03529;
+$grey6: #999999;
+$blue: #41b0ee;
+$darkBlue1: #0092e1;
```

Comment:
can you use more meaningful names like `$darker-blue` and `$darkest-blue`?

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/29/commits/2b668852b23d63790e2f37e7a82001a8673b8ad5